mkdocs.yml

```diff
@@ -4,6 +4,7 @@ nav:
     - Overview: README.md
     - User Guide:
           - Introduction: doc/user_guide.md
+          - ... | src/visualizers-workbench/*
```

Comment:
maybe the weird "src" is coming from here

Reply:
Yes, the 'Src' is a result of the page being in the `src/` subdirectory of the file tree. That is correct.

Reply:
Oh, cool! The workbench and regular docs look the same for me, now. Thanks, @gwhitney.